tests/task/test_standard_task_runner.py

```diff
@@ -84,6 +84,38 @@ def propagate_task_logger():
             h.maintain_propagate = _propagate
 
 
+@pytest.mark.parametrize(["impersonation"], (("nobody",), ("airflow",), (None,)))
+@mock.patch("subprocess.check_call")
+# Mock this to avoid DB calls in render template. Remove this once AIP-44 is removed as it should be able to
+# render without needing ti/DR to exist. Make it so
+@mock.patch("airflow.utils.log.logging_mixin.LoggingMixin._set_context")
```

Comment:
This test before had a mark to use db paccess in test_base_task_runner. Once less test needing DB